src/box/memtx_tx.c

```diff
@@ -2870,25 +2870,30 @@ memtx_tx_tuple_clarify_slow(struct txn *txn, struct space *space,
 	return res;
 }
 
+/**
+ * Run @a code on @a space @a index's top stories.
+ */
+#define memtx_tx_foreach_top_story(space, index, story, code) do {	       \
```

Comment:
Should be updated when #10456 is merged. Please, do not forget to write in the description of the macro that all the chains containing excluded tuples are skipped :)
I labeled this PR as `do not merge` so it won't be accidentally merged before my patch - feel free to drop the label after rebasing :)